src/box/lua/config/instance_config.lua

```diff
@@ -1404,13 +1404,24 @@ return schema.new('instance_config', schema.record({
         }, {
             default = 'off',
         }),
-        -- XXX: needs more validation
         peers = schema.array({
             items = schema.scalar({
                 type = 'string',
             }),
-            box_cfg = 'replication',
             default = box.NULL,
+            validate = function(data, w)
+                if #data == 0 then
+                    w.error('replication.peers can only be nil or non-empty')
+                end
```

Comment:
I think it is OK to temporary disable replication to the given instance during some maintenance task.